ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js

```diff
@@ -1194,6 +1197,9 @@ if (typeof window.DatePickerWidget === 'undefined') {
           case 'focus':
             handler(e);
             break;
+          case 'input':
```

Comment:
Why do you need this listener in the current CL? Please fix this as a separate JIRA

Reply:
PR will not pass if this is not fixed, or will have to comment out a test case